File: code/75. Sort Colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int n = nums.size();
        int a[3] = {0};
         for(int i = 0; i < n ; i++)
         {
         	a[nums[i]]++;
         }
         int j;
         for(j = 0; j < a[0]; j++)
         nums[j] = 0;
         for(j = a[0]; j < a[1] + a[0]; j++)
         	nums[j] = 1;
         for(j = a[0]+a[1]; j < n;j++)
         	nums[j] = 2;
    }
};

File: code/75. Sort Colors.go
func sortColors(nums []int)  {
    r := 0
    w := 0
    b := 0
    for _, num := range nums {
        if num == 0 {
            nums[b] = 2
            b++
            nums[w] = 1
            w++
            nums[r] = 0
            r++
        } else if num == 1 {
            nums[b] = 2
            b++
            nums[w] = 1
            w++
        } else if num == 2 {
            nums[b] = 2
            b++
        }
    }
}